Sources/XcodeGenCLI/Commands/ProjectCommand.swift

```diff
@@ -46,8 +67,31 @@ class ProjectCommand: Command {
             throw GenerationError.projectSpecParsingError(error)
         }
 
+        Logger.shared.debug("Loaded project")
+
         try execute(specLoader: specLoader, projectSpecPath: projectSpecPath, project: project)
     }
 
     func execute(specLoader: SpecLoader, projectSpecPath: Path, project: Project) throws {}
+
+
+    func debug(_ string: String) {
+        stdout.print("[DEBUG] \(string)")
+    }
+
+    func info(_ string: String, wasSuccess: Bool) {
+        var coloredString = string
+        if wasSuccess {
+            coloredString = coloredString.green
+        }
+        stdout.print("[INFO] \(coloredString)")
```

Comment:
This is the only "visible" change of this PR. Instead of seeing:

```
⚙️  Generating plists...
⚙️  Generating project...
⚙️  Writing project...
```

You'll now see:

```
[INFO] ⚙️  Generating plists...
[INFO] ⚙️  Generating project...
[INFO] ⚙️  Writing project...
```